Formula/stubby.rb

```diff
@@ -0,0 +1,93 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver based on getdns"
+  homepage "https://getdnsapi.net"
+  url "https://getdnsapi.net/releases/getdns-1-1-1rc1/getdns-1.1.1rc1.tar.gz"
+  sha256 "f63340b1d05410b875217c6abd7066586fc55a811db4ae90ffd01d2240e05e57"
+  head "https://github.com/getdnsapi/getdns.git", :branch => "develop"
+
+  depends_on "openssl"
+  depends_on "libevent" => :recommended
+  depends_on "libuv" => :optional
+  depends_on "libev" => :optional
+
+  if build.head?
+    depends_on "libtool"
+    depends_on "autoconf"
+    depends_on "automake"
+  end
+
+  def install
+    if build.head?
+      system "glibtoolize", "-ci"
```

Comment:
These are the build instructions lifted from the README:
https://getdnsapi.net/documentation/readme/